### DISCUSSION TITLE
Are property values sets or lists?

### DISCUSSION THREAD

**Discussion by nichtich** [pg-format/pg-formatter#32] [Q&A] [answered]
I thought property values to be sets, so

    a :node x:1 x:1

is equivalent to

    a :node x:1

But apparently Neo4j Cypher accepts

    CREATE (a:node {x:[1,1]})
    
 To create  a node with property key `x` having value list  `1, 1`.

**Comment by hchiba1** [comment, marked as answer]
I treated property values as a list.
I think it is more versatile to treat the values as a list.
(Taking a unique set of values could be done by applications.)
